VRCFaceTracking/Params/Eye/EyeTrackingParams.cs

```diff
@@ -22,7 +22,50 @@ public static class EyeTrackingParams
             new BoolEyeParameter(v2 => v2.Left.Openness < 0.5f, "LeftEyeLid"),
             new BoolEyeParameter(v2 => v2.Right.Openness < 0.5f, "RightEyeLid"),
             new BoolEyeParameter(v2 => (v2.Left.Openness + v2.Right.Openness)/2 < 0.5f, "CombinedEyeLid"),
-            
+
+            // Testing out the bool idea: Ideally would handle this in a class, but a test is a test.
+            new BoolEyeParameter(v2 => ((int)(((v2.Left.Openness * 15 + .5))%2) == 1), "LeftEyeLid1"),
```

Comment:
Could this please be added to a new class for quarter-params? Thanks

Reply:
I will see about that! Are the lip params good to go also?

Reply:
Lip params look good for now! Perhaps I'll refactor them in the future, but for now they look fine!